### PR TITLE
Added the group description support 

### PR DESCRIPTION
**Pull request by Sunilkumarr**
## Summary
Added the group description support now user can edit or add the description 

Group description edit screen : 
Simulator Screen Shot - iPhone 11 - 2021-04-16 at 13 48 47

Group info screen  description will show like this : 
Simulator Screen Shot - iPhone 11 - 2021-04-16 at 13 49 05

In group create description can be added : 
Simulator Screen Shot - iPhone 11 - 2021-04-16 at 15 34 22

## Testing
Tested the description by creating a group adding description and editing the existing description in group

### CODE REVIEW

File: Sources/Models/ALKChannelDetailViewConfiguration.swift [hunk 1]
```diff
@@ -53,5 +53,8 @@ public struct ALKChannelDetailViewConfiguration {
         text: UIColor.black
     )
 
+    /// If true, then group description edit willl be disabled.
```
Comment (mukeshthawani) [inline Sources/Models/ALKChannelDetailViewConfiguration.swift:56]: Typo: 'willl'

File: Sources/Controllers/ALKCreateGroupViewController.swift [hunk 2]
```diff
@@ -448,80 +455,131 @@ extension ALKCreateGroupViewController: ALKCreateGroupViewModelDelegate {
...
     func collectionView(_: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        guard
-            let viewModel = viewModel,
-            let options = viewModel.optionsForCell(at: indexPath.row)
-        else { return }
-        let memberInfo = viewModel.rowAt(index: indexPath.row)
-        let optionMenu = UIAlertController(title: nil, message: memberInfo.name, preferredStyle: .actionSheet)
-        options.forEach {
-            optionMenu.addAction($0.value(localizationFileName: localizedStringFileName, index: indexPath.row, delegate: self))
+        if indexPath.section == CollectionViewSection.groupDescription {
```
Comment (mukeshthawani) [inline Sources/Controllers/ALKCreateGroupViewController.swift:458]: You can use `switch` statement here in place of `if`

File: Sources/Controllers/ALKCreateGroupViewController.swift [hunk 3]
```diff
@@ -448,80 +455,131 @@ extension ALKCreateGroupViewController: ALKCreateGroupViewModelDelegate {
...
+            if addContactMode == .newChat {
+                groupDescriptionVC.delegate = self
+            }
+            navigationController?.pushViewController(groupDescriptionVC, animated: true)
+        } else if indexPath.section == CollectionViewSection.addMemeber {
+            guard
+                let groupName = txtfGroupName.text?.trimmingCharacters(in: .whitespacesAndNewlines),
+                groupName.lengthOfBytes(using: .utf8) > 1
+            else {
+                let msg = localizedString(forKey: "FillGroupName", withDefaultValue: SystemMessage.Warning.FillGroupName, fileName: localizedStringFileName)
```
Comment (mukeshthawani) [inline Sources/Controllers/ALKCreateGroupViewController.swift:472]: Move these `localizedString`s to an `enum` and reduce the code in this method.

Comment (Sunilkumarr) [reply Sources/Controllers/ALKCreateGroupViewController.swift]: Did the changes

File: Sources/Controllers/ALKCreateGroupViewController.swift (hunk 2 above)
Comment (Sunilkumarr) [reply Sources/Controllers/ALKCreateGroupViewController.swift]: changed it to switch

File: Sources/Models/ALKChannelDetailViewConfiguration.swift (hunk 1 above)
Comment (Sunilkumarr) [reply Sources/Models/ALKChannelDetailViewConfiguration.swift]: corrected

**Review by mukeshthawani — APPROVED** [review]

**Pull request merged**